### ISSUE TITLE
comma-spacing doesn’t check arguments of function calls

### ISSUE THREAD

**Issue by lo1tuma** [eslint/eslint#1515]
The following invalid test case fails on current master:

``` js
{
    code: "foo(a,b)",
    args: [2, {before: true, after: true}],
    errors: 2
 }
```
